keps/sig-apps/2879-ready-pods-job-status/README.md

```diff
@@ -131,12 +131,17 @@ pods that have the `Ready` condition.
 #### Alpha
 
 - Feature gate disabled by default.
-- Unit and integration tests passing.
+- Unit and [integration] tests passing.
+
+[integration]: https://testgrid.k8s.io/conformance-all#Conformance%20-%20GCE%20-%20master&include-filter-by-regex=sig-apps&include-filter-by-regex=Job&exclude-filter-by-regex=CronJob
 
 #### Beta
 
 - Feature gate enabled by default.
-- Existing E2E and conformance tests passing.
+- Existing [E2E] and [conformance] tests passing.
```

Comment:
I'd like to see some scalability tests results for beta graduation, since enabling this feature might increase in job status updates and having a clear information what the impact is is an important factor.

Reply:
There's a section `What specific metrics should inform a rollback?` that would be especially important for the above question. The next question `Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?` I'd like to see answered as well.

Reply:
@wojtek-t how do we go about scalability?
I could certainly add an scalability test. We have useful metrics from the job controller too.
Is there a prow job we can reuse?

Reply:
We're testing Jobs as part of our regular e2e scalability load test (not super extensively, but there are Jobs there). 
I'm not sure I would be adding a dedicated CI test for that, but I certainly agree that seeing results from some manual test, that is being "bad case" for this feature would be desired.

Reply:
Added scalability tests (that we need to add) as graduation criteria.